lib/vmpooler/api.rb

```diff
@@ -258,8 +258,10 @@ def mean(list)
             halt 400, 'Invalid "to" date format, must match YYYY-MM-DD'
           end
 
-          # We do need a from date, however.
-          if from_param.nil? or !validate_date_str(from_param.to_s)
+          # It is also ok if from_param is nil, we will assume this also means 'today'
```

Comment:
It allows the param to be unset, and the default `/summary` view (with no params supplied) is a summary of the current day.